extras/docker/centos/prepare-build.sh

```diff
@@ -11,5 +11,6 @@ set -e
 # the local directory where the Dockerfile is located
 WORKDIR=$(dirname "${0}")
 
-cp -f "${WORKDIR}"/../fromsource/heketi.json "${WORKDIR}"
-cp -f "${WORKDIR}"/../fromsource/heketi-start.sh "${WORKDIR}"
+# We used to copy files into the working directory. These files are now
+# included when 'make install' is run.
+true
```

Comment:
now this file does nothing, do we need this file anymore?

Reply:
We can drop it, but the CentOS Container Pipeline still calls out to it. Only once the configuration has been adjusted there, this file can be removed.

Reply:
Understandable. Could we edit the comment to mention that the script is deprecated and/or obsolete and only exists for compatibility with the CentOS Container Pipeline? Perhaps, also mention that the script can be removed once the container pipeline is updated?

Reply:
I'll replace the comments in the script with a clearer explanation.